checkita-core/src/main/scala/ru/raiffeisen/checkita/context/DQJob.scala

```diff
@@ -293,6 +298,23 @@ trait DQJob extends Logging {
       metResults.toSeq.flatMap(_._2).flatMap(r => r.finalizeMetricErrors)
     }
 
+  protected def performJobResult(jobConfig: JobConfig)
```

Comment:
Lets rename this method to `finalizeJobState`. In addition, do not wrap JobState in sequence.